packages/material-ui/src/CircularProgress/CircularProgress.test.js

```diff
@@ -28,118 +27,139 @@ describe('<CircularProgress />', () => {
-      assert.strictEqual(wrapper.hasClass(classes.root), true);
-      const svg = wrapper.childAt(0);
-      const style = svg.childAt(0).props().style;
-      assert.strictEqual(style.strokeDasharray, '126.920', 'should have strokeDasharray set');
-      assert.strictEqual(style.strokeDashoffset, '38.076px', 'should have strokeDashoffset set');
-      assert.strictEqual(wrapper.props()['aria-valuenow'], 70);
+      const { container } = render(<CircularProgress variant="static" value={70} />);
+      const circularProgress = container.firstChild;
+      expect(circularProgress).to.have.class(classes.root);
+      const svg = circularProgress.firstChild;
+      const circle = svg.firstChild;
+      expect(circle.style.strokeDasharray).to.match(
+        /126\.920?(px)?/gm,
+        'should have strokeDasharray set',
+      );
+      expect(circle.style.strokeDashoffset).to.equal(
+        '38.076px',
+        'should have strokeDashoffset set',
+      );
+      expect(circularProgress).to.have.attribute('aria-valuenow', '70');
     });
   });
 
   describe('prop: variant="determinate"', () => {
     it('should render with determinate classes', () => {
-      const wrapper = shallow(<CircularProgress variant="determinate" />);
-      assert.strictEqual(wrapper.hasClass(classes.root), true);
-      const svg = wrapper.childAt(0);
-      assert.strictEqual(svg.name(), 'svg');
-      assert.strictEqual(
-        svg.hasClass(classes.svgIndeterminate),
-        false,
+      const { container } = render(<CircularProgress variant="determinate" />);
+      const circularProgress = container.firstChild;
+      expect(circularProgress).to.have.class(classes.root);
+      const svg = circularProgress.firstChild;
+      expect(svg.tagName).to.equal('svg');
+      expect(svg).to.not.have.class(
+        classes.svgIndeterminate,
         'should not have the svgIndeterminate class',
       );
     });
 
     it('should set strokeDasharray of circle', () => {
-      const wrapper = shallow(<CircularProgress variant="determinate" value={70} />);
-      assert.strictEqual(wrapper.hasClass(classes.root), true);
-      const svg = wrapper.childAt(0);
-      const style = svg.childAt(0).props().style;
-      assert.strictEqual(style.strokeDasharray, '126.920');
-      assert.strictEqual(style.strokeDashoffset, '11.423px');
-      assert.strictEqual(wrapper.props()['aria-valuenow'], 70);
+      const { container } = render(<CircularProgress variant="determinate" value={70} />);
+      const circularProgress = container.firstChild;
+      expect(circularProgress).to.have.class(classes.root);
+      const svg = circularProgress.firstChild;
+      const circle = svg.firstChild;
+      expect(circle.style.strokeDasharray).to.match(
+        /126\.920?(px)?/gm,
+        'should have strokeDasharray set',
+      );
+      expect(circle.style.strokeDashoffset).to.equal(
+        '11.423px',
+        'should have strokeDashoffset set',
```

Comment:
You can use `to.have.property('strokeDashoffset', '11.423px')` in which you don't need to include the property name in the fail message.

Reply:
It seams to not work. I print a debug() and the result is
```html
<circle
   class="MuiCircularProgress-circle MuiCircularProgress-circleStatic"
   cx="44"
   cy="44"
   fill="none"
   r="20.2"
   stroke-width="3.6"
   style="stroke-dasharray: 126.920; stroke-dashoffset: 38.076px;"
/>
```

Reply:
Svg may work differently. this is good as-is.